### ISSUE TITLE
Focus Next Search Result doesn't work due to keyboard shortcut conflict

### ISSUE THREAD

**Issue by gregmarr** [Microsoft/vscode#19850]
- VSCode Version: 1.9.0
- OS Version: OS X 10.11.6

F4 to go to the next search result doesn't work when the focus is in the text editor due to this shortcut key:

```
{ "key": "f4",                    "command": "C_Cpp.SwitchHeaderSource",
                                     "when": "editorTextFocus" },
```

The workaround is to re-add the shortcut in the user settings.

```
{
    "key": "f4",
    "command": "search.action.focusNextSearchResult",
    "when": "editorTextFocus"
}
```

**Comment by roblourens** [comment]
I think they are going to release an update to the cpp extension that removes that binding: https://github.com/Microsoft/vscode-cpptools/issues/458

**Comment by bpasero** [comment]
Reading your description it seems that the issue originates from an extension and not VS Code itself. Thereby I will close this issue and kindly ask you to report this issue against the extension repository. 

Most extensions are on GitHub as well so it should be easy to file it against the extension. If you open an extension on the market place, click on the **Support** link to find out more.

<img width="196" alt="screen shot 2016-05-05 at 12 51 17" src="https://cloud.githubusercontent.com/assets/900690/15041943/263e2282-12c0-11e6-9a33-d2b3b500d61b.png">

Thanks again for your feedback!

**Comment by sean-mcmanus** [comment]
@gregmarr Yeah, the cpptools fix should be in version 0.10.1 we're releasing today.
